src/topology/extremally_disconnected.lean

```diff
@@ -0,0 +1,125 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin
+-/
+import topology.compact_open
+import topology.stone_cech
+
+/-!
+
+# Extremally disconnected spaces
+
+An extremally disconnected topological space is a space
+in which the closure of every open set is open.
+Such spaces are also called Stonean spaces.
+They are the projective objects in the category of compact Hausdorff spaces;
+this fact is proven in `TODO`.
```

Comment:
This and the other occurrence of "proven in `TODO`" are I think a bit confusing. We seem to have proved one way, and the other way should be an explicit TODO in the module docstring, and I'm not sure the fact that this is a TODO should be mentioned in the def docstring.

Reply:
Updated the module docstring and removed the unneeded bit from the def docstring below.  Thanks!